src/makeCalculator.js

```diff
@@ -5,6 +5,33 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate(callback, number) {
+      callback.call(this, number);
+
+      return this;
+    },
+    add(number) {
+      this.result += number;
+    },
+    subtract(number) {
+      this.result -= number;
+    },
+    divide(number) {
+      this.result /= number;
```

Comment:
Ensure that you handle division by zero to avoid potential runtime errors.

Reply:
This should be fixed